README.md

```diff
@@ -210,6 +210,92 @@ This app uses [RSpec](http://rspec.info/) for Rails tests.
 
     rake parallel:spec
 
+## Required environment variables for
 
+* These SMTP settings are used to send emails to prisons. Because these are on the Government Secure Intranet (GSI), they must be sent via MessageLabs. You will need to configured them, if you want to try and test email by submiting a booking request locally. Source: `app/mailers/prison_mailer.rb`
+```ruby
+GSI_SMTP_HOSTNAME
+GSI_SMTP_PORT
+GSI_SMTP_DOMAIN
+```
+
+* These SMTP settings are used to send emails to visitors. Because these using SendGrid's SMTP server, they must be sent via SendGrid. You will need to configured them, if you want to try and test email by submiting a booking request locally. Source: `config/environments/production.rb`
+```ruby
+SMTP_HOSTNAME
+SMTP_PORT
+SMTP_DOMAIN
+SMTP_USERNAME
+SMTP_PASSWORD
+```
+
+* This Service Url that is using for session store the host and links in emails. Though for development and test environment is set to `localhost`. Source: `config/environments/production.rb` && `config/initializers/session_store.rb`
+```ruby
+SERVICE_URL
+```
+
+* The Google Analytics ID. Source: `config/application.rb`
+```ruby
+GA_TRACKING_ID
+```
 
+* The Prison Data File is using the default one `prison_data_staging.yml`. Though you can give a custom name for prison data file. Source: `config/application.rb`
+```ruby
+PRISON_DATA_FILE
+```
+
+* The Prison Estate Ips config.permitted_ips_for_confirmations. Source: `config/application.rb`
+```ruby
+PRISON_ESTATE_IPS
+```
+
+* The Message Encryptor Secret Key is using the default one `key { 'a' * 88 }`.
+Though you can give a custom Message Encryptor Secret Key by setting an 88 char. Source: `config/initializers/encryptor.rb`
+```ruby
+MESSAGE_ENCRYPTOR_SECRET_KEY
+```
+
+* The GovUk Start Page is using the default one `/prisoner`. Though you can give a custom start page. Source: `config/routes.rb`
+```ruby
+GOVUK_START_PAGE
+```
 
+* The Smoke Test for the rails app. Source: `config/environments/development.rb` && `config/environments/production.rb`
+```ruby
+SMOKE_TEST_EMAIL_LOCAL_PART
+SMOKE_TEST_EMAIL_DOMAIN
+```
+
+* The Smoke Test for the `smoke_test`.
+```ruby
+SMOKE_TEST_EMAIL_LOCAL_PART
+SMOKE_TEST_EMAIL_DOMAIN
```

Comment:
Aha, now I understand. I would suggest putting them together, something like this:

```markdown
* Smoke test configuration. Only the first two (local part and domain) are needed in order to
run the application; to run the smoke tests you'll also need the email password and host.
Source: `config/environments/development.rb` and `config/environments`.
```

PS - all those `&&` after 'Source' should probably be and - this is English, not Ruby :smile: